src/playback/config.js

```diff
@@ -122,9 +122,9 @@ const PLATFORMS = [
 
 
 const BROWSERS =[
-  {id: "firefox", label:"Firefox", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-firefox-live"]}}},
-  {id: "geckoview", label:"geckoview", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-geckoview-live"]}}},
-  {id: "fenix", label:"Firefox Preview", filter: {eq: {framework: 10, repo: 'fenix', suite: ["raptor-youtube-playback-fenix-live"]}}},
+  {id: "firefox", label:"Firefox", title:"Firefox (Desktop)", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-firefox-live"]}}},
+  {id: "geckoview", label:"geckoview", title:"Firefox (Android)", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-geckoview-live"]}}},
+  {id: "fenix", label:"Firefox Preview", title:"Firefox Preview (Android)", filter: {eq: {framework: 10, repo: 'fenix', suite: ["raptor-youtube-playback-fenix-live"]}}},
```

Comment:
Good.  I would have added a new property `type: 'mobile'`, which is used elsewhere.